bus_presence_override/static/src/less/systray.less

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2017 LasLabs Inc.
+   License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+#userStatus {
+    &:hover i {
+        color: #d3d3d3 !important;
+    }
+
+    i {
+        padding-top: 2px;
+    }
+}
+
+.o_user_presence_dropdown {
+    min-width: 85px;
+
+    .o_user_presence_status {
+        margin-top: 2px;
+        padding: 4px 8px;
+        width: 100%;
+        display: block;
+        font-size: 13px;
+    }
+
+    &:hover {
+        cursor: pointer;
+    }
+
+}
+
+@media (max-width: @screen-xs-max) {
+
+        .o_user_presence_status {
+            color: #9d9d9d;
```

Comment:
There's a lot of hardcoded colors here, which is bad. Any reason why you didn't use the pre-existing color variables? https://github.com/odoo/odoo/blob/10.0/addons/web/static/lib/bootstrap/less/variables.less#L6